modules/configuration-annotation-processor/src/main/java/org/apache/ignite/internal/configuration/processor/ConfigurationProcessor.java

```diff
@@ -162,9 +169,11 @@ private boolean process0(RoundEnvironment roundEnvironment) {
             // Find all the fields of the schema.
             List<VariableElement> fields = fields(clazz);
```

Comment:
Are these by any chance the same fields that will be present in `ClassWrapper`? If so, I think we can get rid of this.

Reply:
oops, forget to remove this line, thank you